rest-helix/src/main/java/com/github/twitch4j/helix/domain/DropsEntitlementList.java

```diff
@@ -0,0 +1,35 @@
+package com.github.twitch4j.helix.domain;
+
+import org.apache.commons.lang3.StringUtils;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+
+public class DropsEntitlementList {
+
+    /**
+     * The entitlements
+     */
+    private List<DropsEntitlement> data;
```

Comment:
There is no getter to access this

Reply:
Good catch! Added the proper annotations to the class